database/meterdb/db.go

```diff
@@ -125,7 +125,7 @@ func New(
 			methodLabels,
 		),
 	}
-	return meterDB, utils.Err(
+	return meterDB, errors.Join(
```

Comment:
(style) can we assign a new `err` variable before the return statement like we do in the other metrics functions? I know this wasn't changed in this PR